packages/gatsby-plugin-page-creator/src/collection-extract-query-string.ts

```diff
@@ -4,14 +4,15 @@ import { getGraphQLTag } from "babel-plugin-remove-graphql-queries"
 import fs from "fs-extra"
 import traverse from "@babel/traverse"
 import { extractModel } from "./path-utils"
+import reporter from "gatsby-cli/lib/reporter"
```

Comment:
Can we use the reporter from gatsby-node and prop drill it? This won't fix the gatsby-cli import issue. We could also keep the throw and catch somewhere else.